package.json

```diff
@@ -29,7 +29,8 @@
   },
   "dependencies": {
     "dotenv": "^16.0.1",
-    "mixpanel-browser": "^2.43.0"
+    "mixpanel-browser": "^2.43.0",
+    "webpack": "^5.74.0"
```

Comment:
Why webpack is required? What is the use of this?
I don't think it's as ember has its own build tools

Reply:
Webpack is required to run the [ember-bootstrap](https://www.ember-bootstrap.com/)npm package.

Reply:
Can you please send me the link where it is mentioned?

Reply:
It was not mentioned anywhere in the docs, but it was showing error in the console when I was runing ember-bootstrap,
that this package need webpack.

Reply:
Do you have any screenshots of the error? 
If not, can you please recreate it, and send it here?

Reply:
https://media.discordapp.net/attachments/1023150847846273024/1023207675095429171/unknown.png?width=1440&height=330

Error message
As per discussion, newer versions of `ember-auto-import` require webpack